source/crud/tests/unified/bulkWrite-updateOne-sort.yml

```diff
@@ -0,0 +1,125 @@
+description: BulkWrite updateOne-sort
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - _id: 1
```

Comment:
If this YAML was hand-written, feel free to inline small documents like this in the interset of human readability (e.g. `- { _id: 1, x: 11 }`). Likewise for some of the request operation options below (e.g. `filter: { _id: { $gt: 1 } }`).